.buildkite/scripts/steps/quick_checks.ts

```diff
@@ -0,0 +1,147 @@
+/*
```

Comment:
If the intent is to make this available for developers, how do you feel about moving this to the scripts folder?

Reply:
I see what you mean now! `yarn kbn bootstrap` does not initialize the `.buildkite` folder. Good catch!

I can move this to the dev scripts, however, even then, it's not guaranteed, that any of the checks won't use a more contrived typescript script in `.buildkite`. 

I might need to add a step to the `quick_checks.sh` (esp. if `CI=false`) to bootstrap `.buildkite` if it's out of date.

Reply:
I did some reorganization, I still didn't set up a mechanism to ensure `.buildkite/` is bootstrapped in case of the dev runs, that's a todo still.

One thing that I'm not sure about is how/where to store the list of "quick-check" type checks among all the checks we list @ `.buildkite/scripts/steps/checks/`. For now, I've added a file: `.buildkite/scripts/steps/checks/quick_checks.txt`. Alternatively, we can split quick and big checks to separate folders, or duplicate the list in two places (for CI and for the local dev usage).

Reply:
Humm is this a legacy comment Alex? I think all checks run based on dev scripts so I'm not seeing why we would need the .buildkite package.json deps  installed (given thats what you mean by bootstrapped). It should work okay in a dev environment that ran `yarn kbn bootstrap` just like any other dev script (at least based on the current state of your changes)

Reply:
What I meant is, if now (or in the future) there would be a quick check, one that is a bit more complex, and would be written in `typescript` and would rely on some .buildkite pipeline utils, then that script would fail because the `.buildkite` javascript project is not installed.
I understand for now this is not the case, but we also didn't prescribe quick-checks to be shell-only.

Reply:
ah I see. As we run the quick-checks locally through a script that starts with `require('../src/setup_node_env');` I think it will work as expected as the transpilation of those .ts files will be done by that but yeah, we are not sure if clashes will appear or not 👍

Reply:
We usually don't do `require('../src/setup_node_env'); ` in `.buildkite/**/*.ts` scripts, and quick-checks might refer to these too. But it's okay to fix this only when it appears.